app/src/main/res/values/strings.xml

```diff
@@ -280,17 +280,21 @@
     <string name="calling_accept_call">Accept call</string>
     <string name="calling_decline_call">Decline call</string>
     <string name="calling_turn_camera_on_off">Turn camera on/off</string>
+    <string name="calling_flip_camera_on_off">Flip camera</string>
     <string name="calling_turn_speaker_on_off">Turn speaker on/off</string>
-    <string name="label_message_sent_failure">Message could not be sent.</string>
-    <string name="label_try_again">Try again</string>
-    <string name="content_description_ping_everyone">Ping everyone</string>
+    <string name="calling_label_microphone">Microphone</string>
+    <string name="calling_label_camera">Camera</string>
+    <string name="calling_label_speaker">Speaker</string>
+    <string name="calling_label_flip">Flip</string>
```

Comment:
Based on [this figma design](https://www.figma.com/file/UgKehvwIbiIiQ2CCEdsVuf/Android?node-id=9336%3A98703) shouldn't the strings be `allCaps`?

Reply:
Good catch! 🕵️